browser/importer/chrome_profile_lock.h

```diff
@@ -22,11 +22,12 @@ class ChromeProfileLock : public BrowserProfileLock {
   // Returns true if we lock the profile successfully.
   bool HasAcquired() override;
 
- private:
+ protected:
   bool lock_acquired_;
   base::FilePath user_data_dir_;
```

Comment:
Does` user_data_dir_` need to be protected?
I don't see BraveProfileLock has the need to access it

Reply:
Refactored so `user_data_dir_` can be private again.